service/src/main/java/org/kiwiproject/champagne/model/DeployableSystemThreadLocal.java

```diff
@@ -1,20 +1,29 @@
 package org.kiwiproject.champagne.model;
 
-import java.util.Optional;
-
+import lombok.AllArgsConstructor;
+import lombok.Getter;
 import lombok.experimental.UtilityClass;
 
+import java.util.Optional;
+
 @UtilityClass
 public class DeployableSystemThreadLocal {
 
-    private static final ThreadLocal<Long> THREAD_LOCAL = new ThreadLocal<>();
+    @Getter
+    @AllArgsConstructor
+    public class DeployableSystemInfo {
```

Comment:
Would this ever need to be compared, stringified? If so could use `@Value`

Reply:
At this time no